packages/hash/frontend/src/components/hooks/blockProtocolFunctions/useBlockProtocolCreateEntity.ts

```diff
@@ -0,0 +1,69 @@
+import { useMutation } from "@apollo/client";
+import { EmbedderGraphMessageCallbacks } from "@blockprotocol/graph";
+
+import { createEntity as createEntityQuery } from "@hashintel/hash-shared/queries/entity.queries";
+import { useCallback } from "react";
+import {
+  CreateEntityMutation,
+  CreateEntityMutationVariables,
+} from "../../../graphql/apiTypes.gen";
+import { convertApiEntityToBpEntity } from "../../../lib/entities";
+
+export const useBlockProtocolCreateEntity = (
+  accountId: string,
+): {
+  createEntity: EmbedderGraphMessageCallbacks["createEntity"];
+  createEntityLoading: boolean;
+  createEntityError: any;
```

Comment:
Turns out that all the `Loading/Error` returns were unused, apart from `updateEntityLoading`.

I've removed them all except that and left a `@todo` to consider removing `updateEntityLoading`, since half the hooks didn't have the loading/error returns, and they aren't specified in the BP. 

https://github.com/hashintel/hash/pull/752/commits/18f7e6dad25efbd27ec0a909a9739cbe6f68abfa